gufe/components/proteincomponent.py

```diff
@@ -330,6 +332,11 @@ def _from_dict(cls, ser_dict: dict, name: str = ""):
             # put mi_dict back to class
             atom_monomerInfo = Chem.AtomPDBResidueInfo()
             for key, val in atom_mi_dict.items():
+                if key == "MonomerType":
+                    # DANGER: this risks tying serialization to specific
+                    # RDKit versions
+                    val = AtomMonomerType.values[val]
```

Comment:
I'd be curious which values from the enum we're using.  Aren't we only using PDBResidue here?

Reply:
Yeah, I think that's the only one.

Reply:
So PDBResidue is actually the only possible MonomerType (currently), so we're OK to just hardcode to this tbh